sdk/nodejs/provider/index.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
Nit: 2020 for the new files

```suggestion
// Copyright 2016-2020, Pulumi Corporation.
```